config/database.yml

```diff
@@ -57,6 +59,8 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
+  username: postgres
+  password: 8?rSjWDd#J5!a4T
```

Comment:
- [ ] **Optional:** These credentials can be added to the default block so you don't need to repeat them in the dev, test and production blocks.

Reply:
Hi @brytebee, thank you 🙏